cluster/manifests/kube-downscaler/deployment.yaml

```diff
@@ -26,13 +26,14 @@ spec:
       containers:
       - name: downscaler
         # see https://github.com/hjacobs/kube-downscaler/releases
-        image: registry.opensource.zalan.do/teapot/kube-downscaler:0.18
+        image: registry.opensource.zalan.do/teapot/kube-downscaler:20.3.1
         args:
           - --interval=30
           - --exclude-namespaces=kube-system,visibility
-          - --exclude-deployments=kube-downscaler,downscaler,postgres-operator
+          - --exclude-deployments=kube-downscaler
```

Comment:
What if a postgres statefulset is opt-out to scaledown, shouldn't the `postgres-operator` still run in that case?

Reply:
Yes, but I would rather annotate the Postgres Operator deployment itself than adding all deployments as exclusions here. It already has the annotation:

![Screenshot_2020-04-01 postgres-operator (Deployment in default) - Kubernetes Web View](https://user-images.githubusercontent.com/510328/78123665-cd3e5580-740e-11ea-942f-ff0dba12057c.png)

Reply:
Or scratch that, it was only in one cluster. I will add the exclusion back for now.